atomsci/ddm/utils/datastore_functions.py

```diff
@@ -2018,13 +2025,13 @@ def bulk_update_kv(file, client=None, i=0):
         dataset_key=row[1]
         kv_add=row[2]
         print('i =', i, 'dataset_key = ', dataset_key)
-        if type(kv_add) == str:
+        if isinstance(kv_add, str):
             kv_add=string_to_dict(kv_add)
             len_add = len(kv_add)
         else:
             len_add = 0
         kv_del=row[3]
-        if type(kv_del) == str:
+        if isinstance(kv_del, str):
```

Comment:
test?

Reply:
This will need to wait for a later release when the datastore is back up.